### ISSUE TITLE
Cant search 7tv emotes in emotes tab

### ISSUE THREAD

**Issue by hawichii** [Mm2PL/dankerino#118]
**Describe the bug**

 7tv emotes dont appear in search for the emotes tab

**To reproduce**
 
1. search for a 7tv emote
2. it should not be there

**Screenshots**

![image](https://user-images.githubusercontent.com/80816433/149078079-2e02a4c1-fff0-48bf-8b1b-e3340862cbe6.png)
![image](https://user-images.githubusercontent.com/80816433/149078263-6fdfca20-4f2e-4a1b-a82f-015d61a20a8b.png)

**Chatterino version**

Chatterino Nightly + Dankerino patches 2.3.4 (commit 70a37830f, built on )

**Operating system**

Windows 10

**Additional information**

**Comment by zneix** [comment]
Fixed in chatterino7 fork in dbd99e4cd2c93f190b678aa46b7ca8c03b620979, merging our fork's default branch should fix it.

**Comment by Mm2PL** [comment]
Yep, I had a couple of merge conflicts when I tried to merge it and didn't finish it. I'll probably do it today.

12 sty 2022 22:33:08 Paweł ***@***.***>:
